modules/ROOT/pages/installation-prerequisites.adoc

```diff
@@ -17,7 +17,7 @@ Set permissions and a role for your Salesforce API Community Manager organizatio
 +
 This can be any role from the list, but cannot be none.
 +
-If you don't see any roles in the list, refer to the Salesforce knowledge article https://help.salesforce.com/articleView?language=en_US&type=1&mode=1&id=000322715[Add roles to the role hierarchy, role=external, window=_blank] for instructions to create a role in your organization.
+If you don't see any roles in the list, refer to the Salesforce knowledge article https://help.salesforce.com/articleView?language=en_US&type=1&mode=1&id=000322715[Add roles to the role hierarchy, role=external, window=_blank] for instructions to create a role in your organization by entering the name "Salesforce CRM Content User" as a string.
```

Comment:
This is not ok, a Salesforce CRM Content User is a different thing than the role, could we review this?

Reply:
Thank you! Updated per our discussion. Would you please review?